modules/preview-react/_examples/stories/examples/SelectWithFormik.tsx

```diff
@@ -1,13 +1,38 @@
 import * as React from 'react';
 import {useFormik} from 'formik';
-import {Select} from '@workday/canvas-kit-preview-react/select';
+import {Select, useSelectModel} from '@workday/canvas-kit-react/select';
```

Comment:
Maybe we need one from Main, this looks like it lives under `preview-react`, mind adding one for the select in Main?

Reply:
ah I think all examples are under `preview-react/_examples`, even though they have their own section in storybook. Maybe they should get moved to their own place or under the main react folder?